languages/abbreviations/bg.json

```diff
@@ -0,0 +1,68 @@
+{
+    "abbreviations": {
```

Comment:
It might make more sense to make this key a little more descriptive since the folder name is already `abbreviations`.

Reply:
This set of abbreviations is a catch-all for miscellaneous words that don’t play a specific role in a road name or destination. I guess we could call it `miscellaneous`, but `abbreviations` is OK too in my opinion.